mgmt_cli_test.py

```diff
@@ -96,11 +96,62 @@ def test_manager_sanity(self):
         4) test_client_encryption
         :return:
         """
+        self.test_basic_backup()
```

Comment:
it's supposed to be the first one ?, or it's just for the sake of testing it now ?

Reply:
i think the order doesn't really matter, and because this is a fast one, i think it could be the 1st (@roydahan @ShlomiBalalis , objections?)

Reply:
usually a new test would come last, but can also think of a bunch of reasons why the new one should come first. just saying we should give it some thought.

Reply:
FYI @fgelcer @ShlomiBalalis, I'm just waiting to you two to agree on this.

Reply:
@fruch I don't have any particular objections for it to come first